src/features/offer/components/OfferBody/OfferBody.tsx

```diff
@@ -83,49 +96,53 @@ export const OfferBody: FunctionComponent<Props> = ({
         />
 
         {isDesktopViewport ? (
-          <React.Fragment>
-            <OfferCTAButton
-              offer={offer}
-              subcategory={subcategory}
-              trackEventHasSeenOfferOnce={trackEventHasSeenOfferOnce}
-            />
-            <Spacer.Column numberOfSpaces={4} />
-          </React.Fragment>
+          <OfferCTAButton
+            offer={offer}
+            subcategory={subcategory}
+            trackEventHasSeenOfferOnce={trackEventHasSeenOfferOnce}
+          />
         ) : null}
-
-        <OfferAbout offer={offer} />
-      </InfoContainer>
-
-      <OfferPlace offer={offer} isEvent={subcategory.isEvent} />
-
-      {isOfferAMovieScreening ? (
-        <FeatureFlag featureFlag={RemoteStoreFeatureFlags.WIP_ENABLE_NEW_XP_CINE_FROM_OFFER}>
-          <MovieScreeningCalendar offer={offer} subcategory={subcategory} />
-        </FeatureFlag>
+      </MarginContainer>
+
+      {shouldDisplayAboutSection ? (
+        <MarginContainer isDesktopViewport={isDesktopViewport} gap={0}>
+          <OfferAbout
+            offer={offer}
+            metadata={metadata}
+            hasMetadata={hasMetadata}
+            shouldDisplayAccessibilitySection={shouldDisplayAccessibilitySection}
+          />
+        </MarginContainer>
       ) : null}
 
+      <OfferPlace offer={offer} subcategory={subcategory} />
+
       {isDesktopViewport ? (
         <View testID="messagingApp-container-without-divider">
-          <Spacer.Column numberOfSpaces={2} />
           <OfferMessagingApps offer={offer} />
         </View>
       ) : (
-        <SectionWithDivider visible margin testID="messagingApp-container-with-divider">
-          <Spacer.Column numberOfSpaces={2} />
+        <SectionWithDivider visible margin testID="messagingApp-container-with-divider" gap={8}>
           <OfferMessagingApps offer={offer} />
+          <Spacer.Column numberOfSpaces={4} />
         </SectionWithDivider>
       )}
     </Container>
   )
 }
 
-const Container = styled.View({ flexShrink: 1, width: '100%' })
-
-const InfoContainer = styled.View<{ isDesktopViewport?: boolean }>(({ isDesktopViewport }) => ({
-  gap: getSpacing(6),
-  ...(!isDesktopViewport ? { marginHorizontal: getSpacing(6) } : {}),
+const Container = styled.View<{ isDesktopViewport?: boolean }>(({ isDesktopViewport }) => ({
+  flexShrink: 1,
+  width: '100%',
+  gap: isDesktopViewport ? getSpacing(16) : getSpacing(8),
 }))
 
+const MarginContainer = styled(ViewGap)<{ isDesktopViewport?: boolean }>(
+  ({ isDesktopViewport, theme }) => ({
+    ...(!isDesktopViewport ? { marginHorizontal: theme.contentPage.marginHorizontal } : {}),
+  })
```

Comment:
```suggestion
  ({ isDesktopViewport, theme }) => isDesktopViewport ? {} : { marginHorizontal: theme.contentPage.marginHorizontal }
```
J'ai l'impression que c'est plus simple comme ça 👆